sql/moz-fx-data-shared-prod/telemetry_derived/clients_last_seen_v2/query.sql

```diff
@@ -6,6 +6,12 @@ WITH _current AS (
     -- 28 days for each usage criterion as a single 64-bit integer. The
     -- rightmost bit represents whether the user was active in the current day.
     CAST(TRUE AS INT64) AS days_seen_bits,
+    CAST(active_hours_sum > 0 AS INT64) & CAST(
+      COALESCE(
+        scalar_parent_browser_engagement_total_uri_count_normal_and_private_mode_sum,
+        scalar_parent_browser_engagement_total_uri_count_sum
+      ) AS INT64
```

Comment:
Should this be `>=` some number?
```suggestion
      COALESCE(
        scalar_parent_browser_engagement_total_uri_count_normal_and_private_mode_sum,
        scalar_parent_browser_engagement_total_uri_count_sum
      ) >= 1 AS INT64
```

Reply:
> 0 good catch, TY!